packages/employer-api/src/services/claim.service.ts

```diff
@@ -149,6 +154,7 @@ export const addServiceProviderClaim = async (
                     updated_by: submissionResponse.submission.updatedBy ?? submissionResponse.submission.createdBy,
                     updated_date: new Date()
                 })
+                .returning(["catchmentno", "id"])
```

Comment:
I think this line may break the check that occurs on the result in the event controller, since that check expects the result to contain the number of records updated.

Reply:
Oops that's an uncommitted change, should be fixed back to correct now.